src/store/mod.rs

```diff
@@ -110,29 +123,30 @@ impl Store {
         let storage_slots =
             serde_json::to_string(&storage_slots).map_err(StoreError::InputSerializationError)?;
 
-        self.db
-            .execute(
-                "INSERT INTO account_storage (root, slots) VALUES (?, ?)",
-                params![storage_root, storage_slots],
-            )
-            .map(|_| ())
-            .map_err(StoreError::QueryError)
+        tx.execute(
+            "INSERT INTO account_storage (root, slots) VALUES (?, ?)",
+            params![storage_root, storage_slots],
+        )
+        .map(|_| ())
+        .map_err(StoreError::QueryError)
     }
 
-    pub fn insert_account_vault(&self, account_vault: &AccountVault) -> Result<(), StoreError> {
+    fn insert_account_vault(
```

Comment:
It's likely that we want to insert rows for some tables outside of accounts, but for now it should be fine to remove the `pub`s